### PR TITLE
Calculator components

### PR DESCRIPTION
**Pull request by Ibnballo1**
I created the basic calculator components using 3 components
I used a parent component called Calculator.js to hold other components.
Also, I used another component for the result, and lastly, I used a different components for the keypad

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-import './index.css';
-import App from './App';
+import Calculator from './components/Calculator';
+import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```
Comment (tufoinnkuo10) [inline src/index.js]: - [ ] According to the guidlines, the calculator app should be rendered from the app.js component. kindly fix that. this helps you get used to the habit of always displaying your components from here(app.js)
- [ ] please in the process, ensure that the app.js component is class based as well

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.js (hunk 1 above)
Comment (Ibnballo1) [reply src/index.js]: ok. I forgot that. will fix it

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/extensions */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable lines-between-class-members */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import Result from './Result.js';
```
Comment (danmainah) [inline src/components/Calculator.js:9]: - [ ] @Ibnballo1 great work implementing the changes requested but your app is still not running, kindy check how you import your result.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Ibnballo1 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (Ibnballo1) [reply src/components/Calculator.js:9]: My App is working fine...
Kindly recheck

Comment (danmainah) [reply src/components/Calculator.js:9]: @Ibnballo1 am getting this error when running :

image

Comment (Ibnballo1) [reply src/components/Calculator.js:9]: I don't have the issue here

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Ibnballo1     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**